### PR TITLE
feat(widget): APT-986 add props to render inside wallet as dapp

### PR DESCRIPTION
**Pull request by lusd**
* added support for  account and network properties for widget.
* added invalidNetworkModal window. 
* other small UI fixes. 
* added to custom Events to sign and submit transactions.

### CODE REVIEW

File: src/components/Steps/SendTransaction/SubmitTransaction.vue [hunk 1]
```diff
@@ -120,7 +120,7 @@ const isSuccess = computed(() => props.state.status === 'success');
 const { copy: onCopyAddress } = useClipboard();
 const txLink = computed(
   () =>
-    `https://explorer.aptoslabs.com/txn/${txState.value.hash}?network=mainnet `,
+    `https://explorer.aptoslabs.com/txn/${txState.value.hash}?network=mainnet`,
```
Comment (sirWill) [inline src/components/Steps/SendTransaction/SubmitTransaction.vue:123]: network name should correspond to selected network: mainnet/testnet

File: src/components/Steps/SendTransaction/SubmitTransaction.vue [hunk 2]
```diff
@@ -141,6 +141,12 @@ function copyAddress() {
 }
 
 function onComplete() {
+  const event = new CustomEvent('transactionProcessed', {
+    composed: true,
+    bubbles: true,
+  });
+  document.querySelector('liquidswap-widget')!.dispatchEvent(event);
```
Comment (sirWill) [inline src/components/Steps/SendTransaction/SubmitTransaction.vue:148]: `!.` → `?.`

File: src/components/Steps/SendTransaction/SubmitTransaction.vue [hunk 3]
```diff
@@ -141,6 +141,12 @@ function copyAddress() {
...
+
   if (isSuccess.value) {
     // TODO: persist LP?
```
Comment (sirWill) [inline src/components/Steps/SendTransaction/SubmitTransaction.vue:151]: remove comment

File: src/components/CurveSwitch/CurveSwitch.vue [hunk 4]
```diff
@@ -53,10 +53,10 @@ interface IProps {
 
 const props = defineProps<IProps>();
 
-const toast = useToast();
 const { curves } = useStore();
 const store = useSwapStore();
 const tokenStore = useTokensStore();
+// const toast = useToast();
```
Comment (sirWill) [inline src/components/CurveSwitch/CurveSwitch.vue:59]: Why is it commented here, but still in use in code below?

File: src/store/useStore.ts [hunk 5]
```diff
@@ -201,5 +214,10 @@ export const useStore = createGlobalState(() => {
     showDialog,
     modules,
     networkOptions,
+    insideNativeWallet,
+    nativeWalletAccount,
+    nativeNetworkData,
+    nativeStatusTransaction,
+    nativeTransactionHash
   }
 });
```
Comment (sirWill) [inline src/store/useStore.ts:223]: Need to fix

**Review by sirWill — CHANGES_REQUESTED** [review]

File: src/components/Steps/SendTransaction/SubmitTransaction.vue (hunk 1 above)
Comment (lusd) [reply src/components/Steps/SendTransaction/SubmitTransaction.vue:123]: fixed

File: src/components/Steps/SendTransaction/SubmitTransaction.vue (hunk 3 above)
Comment (lusd) [reply src/components/Steps/SendTransaction/SubmitTransaction.vue:151]: fixed

File: src/components/Steps/SendTransaction/SubmitTransaction.vue (hunk 2 above)
Comment (lusd) [reply src/components/Steps/SendTransaction/SubmitTransaction.vue:148]: fixed

File: src/components/CurveSwitch/CurveSwitch.vue (hunk 4 above)
Comment (lusd) [reply src/components/CurveSwitch/CurveSwitch.vue:59]: Yeah, wrong context assignment in main.ts file leed to incorrect working of ToastService. 

fixed

File: src/store/useStore.ts (hunk 5 above)
Comment (lusd) [reply src/store/useStore.ts:223]: renamed and fixed.

**Review by sirWill — APPROVED** [review]

**Pull request merged**